spice/deps/helper_cb.py

```diff
@@ -64,7 +64,6 @@
 if args.clear:
     clipboard.clear()
     clipboard.set_text("")
-    clipboard.store()
```

Comment:
I believe that we should keep this line.
And remove instead of it previous line:
clipboard.set_text("") --- put in to a clipboard empty string. but, it is still a string.

.store() method is like -- "commit changes"